### PR TITLE
Moving from mobx to unstated

### PR DESCRIPTION
**Pull request by vincentaudebert**
Moving to https://github.com/jamiebuilds/unstated in order to reduce package size.

### CODE REVIEW

File: src/Accordion/accordion.container.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+};
+
+class AccordionContainer extends Container<StoreState> {
+    state = {
+        items: [],
+        accordion: true,
+        onChange: () => {},
+    };
+
+    init(accordion: boolean, onChange: Function) {
```
Comment (ryami333) [inline src/Accordion/accordion.container.js]: I was going to suggest refactoring this to:
```
    init({ state }: $SubType<StoreState>) {
    	this.setState(state);
    }
```
But then you've really only just created an alias for 'setState', so you might as well just remove this and use `setState` (or `setOnChange` and `setAccordion`) instead.

File: src/Accordion/accordion.container.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    setExpanded(key: string | number, expanded: boolean) {
+        this.setState({
+            items: this.state.items.map(item => {
+                if (item.uuid === key) {
+                    return {
+                        ...item,
+                        expanded,
+                    };
+                } else if (this.accordion && expanded) {
```
Comment (ryami333) [inline src/Accordion/accordion.container.js]: `this.accordion` needs to be `this.state.accordion`

File: src/Accordion/accordion.js [hunk 3]
```diff
@@ -1,37 +1,47 @@
...
-import { Provider } from 'mobx-react';
-import { createAccordionStore } from '../accordionStore/accordionStore';
+import React, { type ElementProps } from 'react';
+import { Provider, Subscribe } from 'unstated';
+import AccordionContainer from './accordion.container';
 
 type AccordionProps = ElementProps<'div'> & {
     accordion: boolean,
     onChange: Function,
+    accordionStore: ?AccordionContainer,
```
Comment (ryami333) [inline src/Accordion/accordion.js]: accordionStore prop is not nullable.

**Comment by vincentaudebert** [comment]
@ryami333 Exciting times ahead! Looks like we are good to go!!!

**Comment by vincentaudebert** [comment]
Oh ok looks like coveralls is going slightly nuts... I think we can improve the rate by testing this in `accordion-wrapper.js` 
screen shot 2018-04-11 at 3 16 57 pm

**Comment by vincentaudebert** [comment]
I'm really confused by coveralls now........

**Comment by vincentaudebert** [comment]
Back to 100% coverage 💥 !

**Pull request merged**